pkgs/applications/graphics/durdraw/default.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, python3Packages
+, fetchFromGitHub
+, ansilove
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "durdraw";
+  version = "0.20.2";
+
+  src = fetchFromGitHub {
+    owner = "cmang";
+    repo = pname;
```

```suggestion
    repo = "durdraw";
```
See https://github.com/NixOS/nixpkgs/issues/277994.